apparmor.d/profiles-a-f/btop

```diff
@@ -0,0 +1,49 @@
+# apparmor.d - Full set of apparmor profiles
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{,usr/}{,local/}bin/btop
+profile btop @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/consoles>
+
+  capability sys_ptrace,
+
+  network netlink raw,
+
+  signal (send),
+  ptrace (read),
+
+  @{exec_path} mr,
+
+  /etc/fstab r,
+  /etc/passwd r,
+
+  owner @{user_config_dirs}/btop/{,**} rw,
+
+  @{sys}/class/power_supply/ r,
+  @{sys}/class/hwmon/ r,
+  @{sys}/devices/system/cpu/cpufreq/policy[0-9]*/scaling_{cur,min,max}_freq r,
+  @{sys}/devices/virtual/thermal/thermal_zone[0-9]*/ r,
+  @{sys}/devices/virtual/thermal/thermal_zone[0-9]*/hwmon[0-9]*/{,*} r,
+  @{sys}/devices/platform/coretemp.[0-9]*/hwmon/hwmon[0-9]*/{,*} r,
+  @{sys}/devices/pci*/*/*/host[0-9]*/*/*/block/*/*/stat r,
+  @{sys}/devices/virtual/block/dm-[0-9]*/stat r,
+  @{sys}/devices/{pci*,virtual}/{,**/}net/*/statistics/{rx,tx}_bytes r,
+  @{sys}/devices/{pci*,virtual}/{,*/*/}net/*/address r,
+  @{sys}/devices/pci*/*/*/usb3/*/*/*/*/power_supply/hidpp_battery_[0-9]*/{,hwmon[0-9]*/} r,
+
+        @{PROC} r,
+        @{PROC}/loadavg r,
+        @{PROC}/uptime r,
+        @{PROC}/@{pid}/comm r,
+        @{PROC}/@{pid}/cmdline r,
+        @{PROC}/@{pid}/stat r,
+        @{PROC}/@{pid}/io r,
```

Comment:
For a better result with future version of AA. Use `@{pids}` here are these can be any process id, not only the one confined by this profile.